mava/configs/env/scenario/hide_and_seek_5_vs_5.yaml

```
@@ -0,0 +1,17 @@
+team_0:
+  seeker: 5
+team_1:
+  hider: 5
```

Maybe you can do something like this:
```suggestion
teams:
  - name: team_0
    members:
      seeker: 5

  - name: team_1
    members:
      hider: 5
```